release_notes/ocp-4-10-release-notes.adoc

```diff
@@ -1224,6 +1385,56 @@ Previously, pre-flight checks did not account for {rh-openstack-first} resource
 
 * Previously, filtering of virtual tables in the {console-redhat-com} was broken. Consequently, all of the available `nodes` would not appear following a search. This update includes new virtual table logic that fixes the filtering issue in the {console-redhat-com}. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1990255[*BZ#1990255*])
 
+[discrete]
+[id="ocp-4-10-monitoring-bug-fixes"]
+==== Monitoring
+
+* Previously, during {product-title} upgrades, the Prometheus service could become unavailable because either two Prometheus pods were located on the same node or the two nodes running the pods rebooted during the same interval. 
+This situation was possible because the Prometheus pods had soft anti-affinity rules regarding node placement and no pod disruption budget. 
+Consequently, metrics were not collected and rules were not evaluated over a period of time.
```

Comment:
I think we should replace
`pod disruption budget` to `poddisruptionbudgets`

Reply:
changed to `poddisruptionbudget` throughout